apps/example-server/src/main/documents/documents.service.ts

```diff
@@ -99,3 +99,34 @@ export const deleteAll = (id: string, groupId: string) => {
             and(eq(documentsTable.id, id), eq(documentsTable.groupId, groupId)),
         )
 }
+
+export const getFilesByEntityNameAndEntityId = async (
+    entityId: string,
+    entityName: string,
+) => {
+    return db
+        .select({ ...getTableColumns(storageTable) })
+        .from(storageTable)
+        .where(
+            and(
+                eq(storageTable.entityName, entityName),
+                eq(storageTable.entityId, entityId),
+            ),
+        )
```

Comment:
Please move the db calls to the service (document-service in this case) so that we can reuse them if needed.